profiles/common/modules/features/nexteuropa_editorial/src/Config.php

```diff
@@ -21,14 +21,23 @@ class Config extends ConfigBase {
    *
    * @param int $uid
    *    User UID.
+   * @param int $new_group
+   *    The group we're adding to the user.
    *
    * @return bool
    *    TRUE if the user is a team member, FALSE otherwise.
    */
-  public function isEditoralTeamMember($uid = 0) {
+  public function isEditoralTeamMember($uid = 0, $new_group = NULL) {
     $is_team_member = FALSE;
     $account = user_load($uid);
     $groups = og_get_groups_by_user($account, 'node');
+
+    // If in the process of adding a new group, add it since cache may not have
```

Comment:
We could implement a bit of code to reproduce this behavior but I don't see the point. The goal of this method is to determine if a user is active within a specific type of group, and we would lose these two conditions doing it that way.